File: 프로그래머스/2/42578. 의상/의상.cpp
#include <bits/stdc++.h>
using namespace std;
int solution(vector<vector<string>> clothes) {
    int answer = 1;
    map<string,int> m;
    
   for(auto t:clothes){
       m[t[1]]++;
   }
    
    for(auto t:m){
        answer *= (t.second + 1);
    }
    return answer - 1;
}

File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    answer = 1
    closet = {}
    for item, category in clothes :
        if category in closet:
            closet[category].append(item)
        else :
            closet[category] = [item]
    for key in closet :
        answer *= len(closet[key]) + 1
    return answer - 1 

